### DISCUSSION TITLE
Washed out colours

### DISCUSSION THREAD

**Discussion by Throwaway24924** [rustdesk/rustdesk#6454] [General]
### Bug Description

When I connect to my macbook through my linux laptop, the colours look washed out and grey. 

I don't know if this happens on the maxbook screen itself because the display panel is broken. 

My linux laptops display isn't very good quality the colours are naturally a bit dull but Rustdesk is even duller.
However, when I use a different remote desktop tool such as steam remote play (_**still using my linux PC**)_, the colours look more vibrant/accurate. 

If I connect with my iPhone, the colours look fine. 

### How to Reproduce

Connect to macbook from Linux laptop. (either direct IP or through ID. )

### Expected Behavior

Client side should display more accurate/vibrant colours, similar to what is seen when using steam remote play.

### Operating system(s) on local side and remote side

Linux -> OSx

### RustDesk Version(s) on local side and remote side

1.22 - 1.23

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/144624632/4a1d4bd2-2372-4ae7-860f-362c59aa733c)

**Comment by 21pages** [comment]
try nightly both side, and use true color (4:4:4)

**Comment by Throwaway24924** [comment]
> try nightly both side, and use true color (4:4:4)

Just installed nightly and turned on true color. Still having the same issue.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/5246
